test/transmit.js

```diff
@@ -538,10 +538,15 @@ describe('transmission', () => {
             // Use state autoValue function to intercept marshal stage
 
             server.state('always', {
-                autoValue() {
+                async autoValue(request) {
 
-                    client.destroy();
-                    return team.work;               // Continue once the request has been aborted
+                    const close = new Teamwork.Team();
+                    request.raw.res.once('close', () => close.attend());
+
+                    client.destroy();               // Will trigger abort then close. Prior to node v15.7.0 the res close came
+                    await close.work;               // asynchronously after req abort, but since then it comes in the same tick.
```

Comment:
total nit, but the multiline comment to the right here should be above these two lines. as it is, it makes it look like the second line of the comment is regarding the second line of the code which is not the case